### DISCUSSION TITLE
Next-Approute-issue with routegroup after deployment to linux server

### DISCUSSION THREAD

**Discussion by hgnayak82** [vercel/next.js#68198] [App Router]
We implemetend routegroup in our nextjs project, as shown below there is group created "(base)" and we have "/home" inside this folder, this functionality works well in local windows machine but
when we deploy the same codebase in linux server we started getting this error,
we did provide the permission to client folder after analysing further we found out that 
below path with "(base)" is not resolved as linux system is returing the below error
Path error in linux
 
![image](https://github.com/user-attachments/assets/477ffc10-e2d9-4062-9e51-bb62cae6000a)

When we try to acces teh page url we got error as below
__next/static/chunks/app/%5Blocale%5D/(base)/layout.js net::ERR_ABORTED 403 (Forbidden)_
Folder structure
 
![image](https://github.com/user-attachments/assets/d08f06bb-430b-4403-9044-853383a63268)

**Can someone help us to resolve this issue**

**Comment by knarrff** [comment]
Hi @hgnayak82 

The parentheses are interpreted by the shell: you need to escape them like so: ```\(base\)```. I am also quite certain, the ```%5B``` and ```%5D``` should not be encoded like that here, but should be the characters ```[``` and ```]```, although here you will have to do the same: ```\[locale\]```, so in the end you arrive at:
```
cd .next/static/chunks/app/\[locale\]/\(base\)
```
or, because it looks nicer, you enclose everything in double quotes, which also prevents the shell from interpreting something:
```
cd ".next/static/chunks/app/[locale]/(base)"
```

**Reply by hgnayak82** [reply]
After we deploy latest code which has routergroup along with [locale], we r not able to access the path , its returing below error,
Any pointer?

we got the error as 
 GET https://**.*.**.***/_next/static/chunks/app/%5Blocale%5D/(base)/layout.js net::ERR_ABORTED 403 (Forbidden)

we did setup i18-locale route in middleware,

_```
    export function middleware(request) {
      return i18nRouter(request, i18nConfig);
    }
    
    // applies this middleware only to files in the app directory
    export const config = {
      matcher: '/((?!api|static|.*\\..*|_next).*)'
    };
```_

so our folder scruture is as shown below where routergroup (base) is inside [locale] folder and home route is inside (base) folder
![image111](https://github.com/user-attachments/assets/44ee285c-2974-4d39-bad7-808d36db7a40)

![image](https://github.com/user-attachments/assets/cb24e4f4-f921-4cb0-8229-7a60998c7f99)

if we observe the error which is thrown while accessing the url, [locale] is resolved properly but (base) is not resolved so in console it throwed error as below and pag didt get loaded
 GET https://**.*.**.***/_next/static/chunks/app/%5Blocale%5D/(base)/layout.js net::ERR_ABORTED 403 (Forbidden)

**Reply by knarrff** [reply]
When you try to "GET" the URL ```https://.*..***/_next/static/chunks/app/%5Blocale%5D/(base)/layout.js```, do not only escape the ```[]```, but also the ```()```, so make it: ```https://.*..***/_next/static/chunks/app/%5Blocale%5D/%28base%29/layout.js```.
